README.md

```diff
@@ -24,6 +24,11 @@ A first version of the project can be found in the [chis-interoperability](https
 
 ![](./docs/images/bad-client-screen.png) If everything is successful you should see this ![](./docs/images/good-client-screen.png)
 
+## Loading CHT configuration
+1. Go into the `cht-config` directory by running `cd cht-config`.
+1. Run `npm install` to install the dependencies.
```

Comment:
I wasn't able to make this work as per instructions. 
The steps I followed to be able to load the config where:

1. Created a directory outside the interoperability project
2. Copied the content of /interoperability/cht-config to the directory created in 1.
3. Executed cht initialise-project-layout
4. Executed cht --url=[https://user:pass@****.my.local-ip.co](https://medic:****@%2A%2A%2A%2A.my.local-ip.co/) --accept-self-signed-certs